runtime/balrt.h

```diff
@@ -81,6 +85,20 @@ typedef struct {
     GC TaggedPtr *members;
 } TaggedPtrArray;
 
+// We are assuming sizeof(void64_t) = 8
+typedef union {
+    int64_t i;
+    TaggedPtr ptr;
+    double f;
+// pr-todo: should I capitalize? picked this style to match standard types
+} void64_t;
```

Comment:
This is so that list set can be reused for all three 64-bit types. Another option is `listSet` to take `sizeof` and do calculations on `void*` of `GenericArray`.